modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,177 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-network-issues.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-manifest-file-multiple-interfaces-br-ex_{context}"]
+= Creating an additional OVN-Kubernetes `br-ex` bridge network
+
+You can create an additional `br-ex` bridge network that has a different configuration to the default `br-ex` bridge. You must define the additional bridge in the target interface definition of a machine config manifest file.  
```

Comment:
> You can create an additional `br-ex` bridge network that has a different configuration to the default `br-ex` bridge.

suggest adding `br-ex1` as this is a common search-term:

~~~
You can create an additional `br-ex` bridge network: `br-ex1` that has a different configuration to the default `br-ex` bridge.
~~~